src/server.ts

```diff
@@ -91,6 +92,7 @@ export class Server {
     this.server = server;
     const app = new Koa();
 
+    app.use(crossOriginIsolation());
```

Comment:
Is there a chance this change could be breaking in the case where someone is explicitly measuring something where external resources are requested?

In this case, may it be worth putting this change behind a `ServerOpts` where the default is the current behavior (not cross origin isolated), but the headers can be opted in?

Reply:
Good point! I totally didn't think of that. :sweat_smile: 

I think making this disabled by default sort of defeats the purpose of improving the timing accuracy, but I also admit I don't have any evidence that CORP/COEP actually does this (maybe the browser treats `localhost` differently? not sure). That said, having the option at least unblocks `SharedArrayBuffer` and friends.

So sure thing: I added a CLI option, `--cross-origin-isolated`, which is disabled by default. I think further experimentation would be needed to see if it has any effect on timing accuracy. :slightly_smiling_face:

Reply:
I think the benefits of having it (even if behind a flag):
 - Safe roll-out of the option. Making it a non-breaking change.
 - The timers _should_ be more accurate when cross origin isolated is enabled.
 - And we could still add the `SharedArrayBuffer` (and friends) features. They would just have to check the [`crossOriginIsolated` property](https://developer.mozilla.org/en-US/docs/Web/API/Window/crossOriginIsolated).

I'm a fan of this change!